src/util/logger.js

```diff
@@ -0,0 +1,11 @@
+import videojs from 'video.js';
+
+const logger = (source) => {
```

Comment:
This will not allow us the ability to turn debug logging on mid session via `videojs.log.level('debug')` . If we want to support that, we will need something like this

```
const logger = (source) => (...args) => {
  if (videojs.log.level() === 'debug') {
    videojs.log.debug.call(videojs, 'VHS:', `${source} >`, ...args);
  }
};
```

Reply:
you don't even need the `.call(videojs,` in your way @mjneil.

Reply:
oh right, good call!

Reply:
oops this is checking if debug api is available, not if the mode is debug